### ISSUE TITLE
Upgraded to 1.1.0 and having extension plugin issues.

### ISSUE THREAD

**Issue by afonit** [jupyterlab/jupyterlab#7121]
I run several extensions as a part of my workflow - after upgrading to jupyter lab 1.1.0 the extension manager is having issues, here is a screenshot:

![image](https://user-images.githubusercontent.com/1798497/63968518-f4463a80-ca6d-11e9-9b5a-43899b0791ae.png)

When I launch jupyter lab the terminal shows this:
```
[E 14:40:41.370 LabApp] 500 GET /lab/api/extensions?1567104038372 (::1) 570.47ms referer=http://localhost:8888/lab
[E 14:40:41.939 LabApp] Uncaught exception GET /lab/api/extensions?1567104038437 (::1)
    HTTPServerRequest(protocol='http', host='localhost:8888', method='GET', uri='/lab/api/extensions?1567104038437', version='HTTP/1.1', remote_ip='::1')
    Traceback (most recent call last):
      File "C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\lib\site-packages\tornado\web.py", line 1699, in _execute
        result = await result
      File "C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\lib\site-packages\tornado\gen.py", line 742, in run
        yielded = self.gen.throw(*exc_info)  # type: ignore
      File "C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\lib\site-packages\jupyterlab\handlers\extension_manager_handler.py", line 226, in get
        extensions = yield self.manager.list_extensions()
      File "C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\lib\site-packages\tornado\gen.py", line 735, in run
        value = future.result()
      File "C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\lib\site-packages\tornado\gen.py", line 209, in wrapper
        yielded = next(result)
      File "C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\lib\site-packages\jupyterlab\handlers\extension_manager_handler.py", line 81, in list_extensions
        build_check_info = _build_check_info(self.app_dir, self.log)
      File "C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\lib\site-packages\jupyterlab\handlers\extension_manager_handler.py", line 55, in _build_check_info
        messages = handler.build_check(fast=True)
      File "C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\lib\site-packages\jupyterlab\commands.py", line 674, in build_check
        new_package = self._get_package_template(silent=fast)
      File "C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\lib\site-packages\jupyterlab\commands.py", line 1121, in _get_package_template
        compat_errors = self._get_extension_compat()
      File "C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\lib\site-packages\jupyterlab\commands.py", line 1255, in _get_extension_compat
        compat[name] = _validate_compatibility(name, deps, core_data)
      File "C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\lib\site-packages\jupyterlab\commands.py", line 1781, in _validate_compatibility
        overlap = _test_overlap(core_deps[key], value)
    KeyError: '@phosphor/domutils'
[W 14:40:41.943 LabApp] Unhandled error
[E 14:40:41.947 LabApp] {
      "Host": "localhost:8888",
      "Connection": "keep-alive",
      "Sec-Fetch-Mode": "cors",
      "Authorization": "token 3a92d5ad206e529b67996bfa612df1ccf34a36b12ea3ed56",
      "X-Xsrftoken": "2|d73a6ba6|22dee716e402131e3d5a6f4ab6318709|1565634955",
      "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3809.132 Safari/537.36",
      "Content-Type": "application/json",
      "Accept": "*/*",
      "Sec-Fetch-Site": "same-origin",
      "Referer": "http://localhost:8888/lab",
      "Accept-Encoding": "gzip, deflate, br",
      "Accept-Language": "en-US,en;q=0.9,de;q=0.8",
      "Cookie": "_xsrf=2|d73a6ba6|22dee716e402131e3d5a6f4ab6318709|1565634955; username-localhost-8888=\"2|1:0|10:1567104037|23:username-localhost-8888|44:N2I1ODU2OTBiZThjNGQ0OTk2OTNhOTVjZmVkMTE4MjY=|7d550846441258a680369855a9c4ca45de22edae01dd982e3eb98958a0ef32d4\""
    
```

Here are my versions:
```
# Name                    Version                   Build  Channel
jupyter                   1.0.0                    py37_7
jupyter_client            5.3.1                      py_0
jupyter_console           6.0.0                    py37_0
jupyter_core              4.5.0                      py_0
jupyterlab                1.1.0                      py_0    conda-forge
jupyterlab_server         1.0.0                      py_1

# Name                    Version                   Build  Channel
nodejs                    12.8.1                        0    conda-forge

```

**Comment by jasongrout** [comment]
I'm trying to reproduce this. It looks like it might be the same as https://github.com/jupyterlab/jupyterlab/issues/7115 and #7116.

**Comment by afonit** [comment]
Last time upgrading to 1.0 I had major issue and had to just start from scratch in a new environment.

If no one else is having this specific issue I will just rebuild everything in a new environment - but thought I should report it in case there are others who would run into this.

**Comment by jasongrout** [comment]
At very least, do a `jupyter lab clean` and build again.

**Comment by afonit** [comment]
I did the clean:
```
[LabCleanApp] Cleaning C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\share\jupyter\lab...
[LabCleanApp] Success!
```

Then started jupyterlab and same issues as in the top post.

**Comment by jasongrout** [comment]
>  You did mention to 'build again', how is that triggered?

`jupyter lab build`

**Comment by afonit** [comment]
ah, thanks, that produces the below:

```
jupyter lab build
[LabBuildApp] JupyterLab 1.1.0
[LabBuildApp] Building in C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\share\jupyter\lab
[LabBuildApp] Building jupyterlab assets (build:prod:minimize)
An error occured.
KeyError: '@phosphor/domutils'
See the log file for details:  C:\Users\JTHOMP~1\AppData\Local\Temp\jupyterlab-debug-jpbm8yxl.log
```

The mentioned log in the above directory shows:
```
[LabBuildApp] Building in C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\share\jupyter\lab
[LabBuildApp] Node v12.8.1

[LabBuildApp] Building jupyterlab assets (build:prod:minimize)
[LabBuildApp] Traceback (most recent call last):

[LabBuildApp]   File "C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\lib\site-packages\jupyterlab\debuglog.py", line 47, in debug_logging
    yield

[LabBuildApp]   File "C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\lib\site-packages\jupyterlab\labapp.py", line 96, in start
    core_config=self.core_config)

[LabBuildApp]   File "C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\lib\site-packages\jupyterlab\commands.py", line 378, in build
    command=command, clean_staging=clean_staging)

[LabBuildApp]   File "C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\lib\site-packages\jupyterlab\commands.py", line 564, in build
    clean=clean_staging

[LabBuildApp]   File "C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\lib\site-packages\jupyterlab\commands.py", line 1070, in _populate_staging
    data = self._get_package_template()

[LabBuildApp]   File "C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\lib\site-packages\jupyterlab\commands.py", line 1121, in _get_package_template
    compat_errors = self._get_extension_compat()

[LabBuildApp]   File "C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\lib\site-packages\jupyterlab\commands.py", line 1255, in _get_extension_compat
    compat[name] = _validate_compatibility(name, deps, core_data)

[LabBuildApp]   File "C:\Users\jthomp261\AppData\Local\Continuum\anaconda3\envs\data_analysis_lab\lib\site-packages\jupyterlab\commands.py", line 1781, in _validate_compatibility
    overlap = _test_overlap(core_deps[key], value)

[LabBuildApp] KeyError: '@phosphor/domutils'

[LabBuildApp] Exiting application: JupyterLab
```

**Comment by jasongrout** [comment]
What is `jupyter labextension list`? Does it work in a clean environment (see https://jupyterlab.readthedocs.io/en/stable/getting_started/issue.html for instructions for doing more diagnostic work like this to help us move forward)?

**Comment by jtpio** [comment]
Saw this one too today (same error).

Reproducible in a clean environment and the installing `jupyterlab_vim` extension: 

![domutils](https://user-images.githubusercontent.com/591645/63973649-799c1000-caab-11e9-86c7-ec1ef86c1408.gif)

**Comment by jasongrout** [comment]
Thanks. That reproduces it for me. Investigating...

**Comment by jasongrout** [comment]
Let's consolidate this conversation over to #7115, which is the same issue, but opened a bit earlier.

**Comment by jasongrout** [comment]
Fix is in jlab 1.1.1, just released on pypi and in process of going up on conda-forge
